recipe/meta.yaml

```diff
@@ -49,10 +49,12 @@ requirements:
     - {{ compiler('fortran') }}              # [not win]
     - {{ compiler('m2w64_fortran') }}        # [win]
     - cmake
-    - {{ mpi }}  # [mpi != 'nompi']
+    - openmpi                                # [mpi == "openmpi" and (build_platform != target_platform)]
```

Comment:
I wonder why is openmpi treated specially here?

Reply:
nvm, only when cross compiling

Reply:
Because in openmpi, the compiler wrappers are binaries and the wrappers in build can use host libraries by adding `OPAL_PREFIX` and in mpich, compiler wrappers are bash scripts and wrappers in build can't use host libraries.

Reply:
Thanks, @isuruf, you have my thumbs up 👍

Reply:
Awesome, I now applied the same OpenMPI work-arounds to fix https://github.com/conda-forge/openpmd-api-feedstock/pull/66 :tada: